### PR TITLE
Added PowerShell examples for Connect-MicrosoftTeams with AccessTokens

### PR DESCRIPTION
**Pull request by mariussm**

### CODE REVIEW

**Comment by yogkumgit** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks

**Review by dariomws — COMMENTED** [review]
@mariussm, thanks a lot for your contribution. Anyway, I think there are several options to acquire an access token and several articles with examples. I understand there is no need to add this information here but let's wait that my colleagues to share their thoughts.

About ApplicationId parameter, I've verified that it's not available in versions 3.0.0 and 2.6.0, so I think we need to confirm from what version it's not available or just remove it.

**Comment by mariussm** [comment]
@dariomws

> @mariussm, thanks a lot for your contribution. Anyway, I think there are several options to acquire an access token and several articles with examples. I understand there is no need to add this information here but let's wait that my colleagues to share their thoughts.
> 
> About ApplicationId parameter, I've verified that it's not available in versions 3.0.0 and 2.6.0, so I think we need to confirm from what version it's not available or just remove it.

In that case we should point to that documentation (which I cannot find). I spent quite a lot of time to figure out the correct way, as the scope url needed the guid instead of the url of the API, and stuff.

**Comment by mariussm** [comment]
Invoke-RestMethod changes to -Method Post automatically when a body is provided, in newer versions of PowerShell 7. :)

**Comment by get-itips** [comment]
@mariussm Marius, it doesn't hurt to add it then, as people could try your example using Windows PowerShell, thanks

**Comment by tseward** [comment]
Another question I'd have is would it be easier to mix this with the MgGraph module (i.e., https://docs.microsoft.com/en-us/graph/powershell/app-only?tabs=azure-portal) rather than hand roll the POST?

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 1]
```diff
@@ -83,9 +83,29 @@ user@contoso.com        TeamsGCCH   xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  xxxxxx
 ### Example 4: Connect to MicrosoftTeams using Accesstokens
 This example demonstrates how to sign in using AccessTokens. Admin can reterive Access Tokens. It requires two tokens, MS Graph Access Token and Teams Resource token.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:84]:
```suggestion
This example demonstrates how to sign in using AccessTokens. Admin can retrieve Access Tokens. It requires two tokens, MS Graph Access Token and Teams Resource token. 
```

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 2]
```diff
@@ -83,9 +83,29 @@ user@contoso.com        TeamsGCCH   xxxxxxxx-xxxx-xxxx-xxxx-xxxxxxxxxxxx  xxxxxx
 ### Example 4: Connect to MicrosoftTeams using Accesstokens
 This example demonstrates how to sign in using AccessTokens. Admin can reterive Access Tokens. It requires two tokens, MS Graph Access Token and Teams Resource token. 
 
+A delegated flow, such as Resouce Owner Password Credentials (ROPC), must be used, with the following delegated app permissions required.
```
Comment (JohanFreelancer9) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:86]:
```suggestion
A delegated flow, such as Resource Owner Password Credentials (ROPC), must be used, with the following delegated app permissions required.
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - @mariussm, kindly commit my suggested changes to correct the typos. I'll do a final edit once the other reviewers' comments have been resolved. Many thanks.

**Comment by yogkumgit** [comment]
@tseward @get-itips @dariomws

File: teams/teams-ps/teams/Connect-MicrosoftTeams.md [hunk 3]
```diff
@@ -156,7 +176,7 @@ Accept pipeline input: False
 Accept wildcard characters: False
 ```
 
-### -ApplicationId
+### -ApplicationId (Removed from latest version)
```
Comment (get-itips) [inline teams/teams-ps/teams/Connect-MicrosoftTeams.md:179]: Can you please move what you added in parenthesis to the description?

**Comment by tiburd** [comment]
@rajgodwin 
Please review this PR, or point us to someone who can.
Thank you!

**Comment by tiburd** [comment]
@tseward 
Hi,

Does this PR have your approval?

Thanks!

**Review by tseward — APPROVED** [review]

**Comment by yogkumgit** [comment]
@get-itips @dariomws

**Pull request merged**